recipes/superflore/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "superflore" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/ros-infrastructure/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: e10bde2a62aec7e2ba8c5040e2d0593a06a99cdc018ccbb58c46f7659705bc02
+
+build:
+  number: 0
+  entry_points:
+    - superflore-gen-ebuilds = superflore.generators.ebuild:main
+    - superflore-gen-oe-recipes = superflore.generators.bitbake:main
+    - superflore-check-ebuilds = superflore.test_integration.gentoo:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  build:
+    - python >=3
+  host:
+    - pip
+    - python
+    - setuptools
+  run:
+    - pywin32  # [win]
+    - beautifulsoup4
+    - catkin_pkg >=0.4.0
+    - docker-py
+    - gitpython
+    - pygithub
+    - python >=3
```

Comment:
```suggestion
    - python
```

Reply:
There are two ways of building packages that use Python in conda-forge:
* **noarch: python**: Use the same "binaries" for all Python distributions and platforms (e.g. a pure Python package). The host/runtime requirements should then constrained using `python >=X.Y`
* **others**: One "binary" per platform and Python version. Constraining the Python version should then be done with `skip: true  # [py<X.Y]`.

As this has different dependencies on Windows it can't use `noarch: python` and has to constrain Python the version the other way.